drivers/include/lsm6ds3.h

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    driver_LSM6DS3 LSM6DS3 3D accelerometer/gyroscope
+ * @ingroup     drivers
+ * @brief       Device driver for the LSM6DS3 3D accelerometer/gyroscope
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for the LSM6DS3 3D accelerometer/gyroscope
+ *
+ * @author      René Herthel <rene-herthel@outlook.de>
```

Comment:
Depends if there is any of Thomas actual code left. If you just use the file as template, I think the common practice was to put in only your own name as author and treat the file as if you have created it. If there is however some code you leave from the original, you should leave in the original author(s).